### PR TITLE
[Common] Add mainfest tests

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

**Review by ayefimov-1 — APPROVED** [review]
Looks good.

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -6,3 +6,10 @@
     - common_pod_status_str is defined
   ansible.builtin.include_tasks: "pod_tests.yml"
   loop: "{{ common_pod_list }}"
+
+- name: "Run mainfest tests"
```
Comment (vyzigold) [inline roles/common/tasks/main.yml:10]:
```suggestion
- name: "Run manifest tests"
```

File: roles/common/tasks/main.yml [hunk 2]
```diff
@@ -6,3 +6,10 @@
...
+  when:
+    - manifest_test_id is defined
+    - manifest_list is defined
+  ansible.builtin.include_tasks: "mainfest_tests.yml"
```
Comment (vyzigold) [inline roles/common/tasks/main.yml:14]:
```suggestion
  ansible.builtin.include_tasks: "manifest_tests.yml"
```

File: roles/common/tasks/mainfest_tests.yml
Comment (vyzigold) [inline roles/common/tasks/mainfest_tests.yml:1]: This should be renamed to "manifest_tests.yml"

**Review by vyzigold — APPROVED** [review]

**Pull request merged**